### ISSUE TITLE
Segfault using column families

### ISSUE THREAD

**Issue by winstonewert** [bh1xuw/rust-rocks#14]
The following code segfaults:

```
use rocks::rocksdb::*;
fn main() {
    let database = DB::open_with_column_families(
        &Options::default_instance(),
        "foobar",
        vec![
            ColumnFamilyDescriptor::new(
                "default",
                ColumnFamilyOptions::new()
            ),
        ]
    ).unwrap();
}
```

The problem is that the database gets dropped before the column factory handles.

**Comment by bh1xuw** [comment]
Thanks, lifetime leaks.
Fixed in v0.1.6.

**Comment by bh1xuw** [comment]
Already fixed.
Feel free to re-open.
